docs/samples.md

```diff
@@ -65,9 +67,11 @@ The following list describes the samples available in the repository:
 * `query_permissions.py` Queries permissions of a given resource.
 * `refresh.py` Refreshes a datasource or workbook.
 * `refresh_tasks.py` Lists and runs configured tasks on a server.
-* `set_http_options.py` Demonstrates HTTP options for the server and specifically for downloading workbooks.
 * `set_refresh_schedule.py` Sets the schedule to refresh a workbook or datasource.
+* `smoke_test.py` Verifies that TSC is installed correctly and is able to run.
 * `update_connection.py` Updates and embeds connection credentials of a datasource.
 * `update_datasource_data.py` Updates the data within a published live-to-Hyper datasource.
+* `update_workbook_data_acceleration.py` Updates workbook data acceleration settings.
```

Comment:
We should probably delete this one, the feature was removed in 2022. https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_concepts_data_acceleration.htm

Reply:
Good catch! Will remove it here and in my samples code PR.